src/Base/ProgressIndicator.h

```diff
@@ -0,0 +1,37 @@
+#ifndef BASE_PROGRESSSCOPE_H
+#define BASE_PROGRESSSCOPE_H
+
+#include <memory>
+
+namespace Base
+{
+
+class ProgressIndicator
```

Comment:
This fails on windows according to CI.
You need to add BaseExport for symbols to be exported on windows. This applies to both classes and static functions.
```suggestion
#include "FCGlobal.h"

namespace Base
{

class BaseExport ProgressIndicator
```

Reply:
Thanks. I've also put GuiExport on ComputationDialog and PartExport on OCCTProgressIndicator, is that correct?